src/libsystemd/sd-netlink/netlink-util.c

```diff
@@ -161,6 +161,13 @@ bool rtnl_message_type_is_addr(uint16_t type) {
         }
 }
 
+bool rtnl_message_type_is_iprule(uint16_t type) {
+        if (IN_SET(type, RTM_NEWRULE, RTM_DELRULE, RTM_GETRULE))
+                return true;
+
+        return false;
```

Comment:
I'd prefer `return IN_SET(...);`, and maybe even make it an inline function in netlink-util.h?